src/main/java/com/google/api/generator/gapic/protoparser/Parser.java

```diff
@@ -663,6 +697,16 @@ private static boolean isMapType(Descriptor messageDescriptor) {
                       /* protoPakkage */ protoMethod.getFile().getPackage(),
                       serviceDescriptor.getName(),
                       protoMethod.getName());
+      boolean operationPollingMethod = false;
+      if (protoMethod.getOptions().hasExtension(ExtendedOperationsProto.operationPollingMethod)) {
```

Comment:
consider using ternary operator (`?`) for stuff like this (this is basically its mos common usage pattern).